### ISSUE TITLE
Undefined array key "class" 

### ISSUE THREAD

**Issue by alberto56** [mglaman/phpstan-drupal#273]
Hi,

I'm getting:

    Warning: Undefined array key "class" in /var/www/html/vendor/mglaman/phpstan-drupal/src/Drupal/ServiceMap.php on line 41

but no errors, and an exit code of zero.

Is this an error with my code? Or a bug with phpstan-drupal, 

thanks!

**Comment by alberto56** [comment]
You can reproduce this by running:

    git clone https://github.com/dcycle/starterkit-drupal8site.git
    cd starterkit-drupal8site 
    ./scripts/static.sh 

The result is:

```

Performing static analsys
If you are getting a false negative, use:

// @phpstan-ignore-next-line

Warning: Undefined array key "class" in /var/www/html/vendor/mglaman/phpstan-drupal/src/Drupal/ServiceMap.php on line 41
 5/5 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%


 [OK] No errors                                                                 
```

**Comment by mglaman** [comment]
Are you using the latest phpstan-drupal? This was being caused by the addition of auto wiring test fixtures in Drupal core.

**Comment by mglaman** [comment]
Please check the `dcycle/phpstan-drupal` image, it probably has outdated dependencies. Feel free to reopen if that is not the case.

**Comment by alberto56** [comment]
Thank you @mglaman I was requiring ^0.12.1 of phpstan-drupal, I changed that to * and now it works.

https://github.com/dcycle/docker-phpstan-drupal/pull/11/files
